services/user/Branding/service/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+#[psibase::service(name = "branding")]
+mod service {
+    use async_graphql::{Object, SimpleObject};
+    use psibase::{
+        anyhow, check, get_sender, get_service, serve_content, serve_graphiql, serve_graphql,
+        serve_simple_ui, store_content, Fracpack, HexBytes, HttpReply, HttpRequest, SingletonKey,
+        Table, ToSchema, WebContentRow,
+    };
+    use serde::{Deserialize, Serialize};
+
+    #[table(name = "NetworkNameTable")]
+    #[derive(Fracpack, ToSchema, SimpleObject, Serialize, Deserialize, Debug)]
+    pub struct NetworkName {
+        pub name: String,
+    }
+
+    impl NetworkName {
+        #[primary_key]
+        fn by_key(&self) -> SingletonKey {
+            SingletonKey {}
+        }
+    }
+
+    impl Default for NetworkName {
+        fn default() -> Self {
+            NetworkName {
+                name: String::from("psibase"),
+            }
+        }
+    }
+
+    #[table(record = "WebContentRow", index = 1)]
+    struct WebContentTable;
+
+    #[action]
+    #[allow(non_snake_case)]
+    fn networkNameChanged(name: String) {
```

Comment:
Suggested tense convention:
* action: imperative (`setNetworkName`)
* event: present perfect (`networkNameChanged`)

Reply:
Can you explain marking this as resolved? The tense still seems strange to me.

Reply:
You're not looking at the most recent code? The diff shows the changes.